ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OGridSearch.scala

```diff
@@ -87,6 +87,7 @@ class H2OGridSearch(override val uid: String) extends Estimator[H2OMOJOModel]
     algoParams._nfolds = getNfolds()
     algoParams._fold_column = getFoldCol()
     algoParams._response_column = getLabelCol()
+    algoParams._weights_column = getWeightCol()
```

Comment:
@jakubhava Please can you confirm that the weight column is also relevant for GridSearch?

Reply:
Yup, this is fine